bin/oref0-setup.sh

```diff
@@ -1115,17 +1132,30 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
           wget -qO- $downloadUrl | tar xJv -C $HOME/go/bin || die "Couldn't download and extract Go pump binaries"
           echo "Installing Go pump binaries ..."
           ln -sf $HOME/go/bin/openaps.jq $directory/ || die "Couldn't softlink openaps.jq"
-          rsync -rtuv $HOME/go/bin/ /usr/local/bin/ || die "Couldn't rsync go/bin"
+          cp -pruv $HOME/go/bin/* /usr/local/bin/ || die "Couldn't copy go/bin"
```

Comment:
What's the advantage of using cp over rsync?

Reply:
consistency, cp is more universal, I had a commit a long while back and we changed other rsync's over to cp